### PR TITLE
fix faucet + removed placeholder wallet pem file

### PR DESCRIPTION
**Pull request by bogdan-rosianu**
Removed `walletKey.pem` placeholder file and fixed the situation when the pem file was missing.

### CODE REVIEW

File: faucet/privateKeysLoader.go [hunk 1]
```diff
@@ -33,8 +33,8 @@ func NewPrivateKeysLoader(
 	if shardCoord == nil {
```
Comment (miiu96) [inline faucet/privateKeysLoader.go:33]: check if nil

File: faucet/privateKeysLoader.go [hunk 2]
```diff
@@ -33,8 +33,8 @@ func NewPrivateKeysLoader(
...
 		return nil, ErrNilShardCoordinator
 	}
-	if len(pemFileLocation) == 0 {
-		return nil, ErrInvalidPemFileLocation
+	if !core.DoesFileExist(pemFileLocation) {
```
Comment (iulianpascalau) [inline faucet/privateKeysLoader.go:36]: suggestion to have core.FileExists. Can be a Jira task for this in the refactoring user story

**Review by iulianpascalau — APPROVED** [review]

File: faucet/privateKeysLoader.go (hunk 2 above)
Comment (bogdan-rosianu) [reply faucet/privateKeysLoader.go:36]: created EN-10032

File: faucet/privateKeysLoader.go (hunk 1 above)
Comment (bogdan-rosianu) [reply faucet/privateKeysLoader.go:33]: done

**Review by miiu96 — APPROVED** [review]

**Review by iulianpascalau — APPROVED** [review]

**Review by raduchis — APPROVED** [review]

**Pull request merged**